dev-docs/modules/userId.md

```diff
@@ -72,7 +72,7 @@ of sub-objects. The table below has the options that are common across ID system
 ## User ID Sub-Modules
 
 ## Permissions
-Publishers can control which user ids are shared with the bid adapters they choose to work with by using the bidders array.  The bidders array is part of the User id module config, publisher may choose to send an id to some bidders but not all, the default behavior is that each user id go to all bid adapters the publisher is working with. 
+Publishers can control which user ids are shared with the bid adapters they choose to work with by using the bidders array.  The bidders array is part of the User id module config, publisher may choose to send an id to some bidders but not all, the default behavior is that each user id go to all bid adapters the publisher is working with.
```

Comment:
why are you editing the permissions description?  Permissions pertains to all sub adapters.